conan/cli/commands/list.py

```diff
@@ -114,7 +114,7 @@ def list(conan_api: ConanAPI, parser, *args):
             results[name] = {"error": str(e)}
         else:
             results[name] = list_bundle.serialize() if args.format in ("json", "html") \
-                else list_bundle.recipes
+                else list_bundle.ordered_recipes_by_name
```

Comment:
I think it is better to leave the ``list_bundle.recipes`` and have that method/property to return the references ordered?